storage/innobase/srv/srv0srv.cc

```diff
@@ -1421,6 +1422,31 @@ srv_export_innodb_status(void)
 		btr_scrub_total_stat(&scrub_stat);
 	}
 
+#ifdef BTR_CUR_HASH_ADAPT
+	mem_adaptive_hash = 0;
```

Comment:
I think that this one should be
```c
ulong mem_adaptive_hash = 0;
```
(remove the earlier declaration, and declare it only if it is being used).